jkamz/metro-comments
# #163356831  user signup

### What does this PR do?
This PR creates the endpoint for signing up a user

### Description of the tasks to be completed

create an endpoint for user sign up

### How can this be manually tested?
Clone this repository on local machine and CD into project folder

`$ git clone https://github.com/jkamz/metro-comments`

`$ cd metro-comments`

Check out to the PR's branch
`$ git checkout ft-user-signup-163356831`


### What are the relevant Pivotal Tracker stories?
#163356831
